coderd/database/queries/templateversions.sql

```diff
@@ -110,3 +110,17 @@ SET
 	updated_at = $3
 WHERE
 	job_id = $1;
+
+-- name: GetPreviousTemplateVersion :one
+SELECT
+	*
+FROM
+	template_versions
+WHERE
+	created_at < (
+		SELECT created_at
+		FROM template_versions as tv
```

Comment:
`AS` is optional, but if we have it, let's be consistent 😄

```suggestion
		FROM template_versions AS tv
```